src/default_parameters.jl

```diff
@@ -133,15 +134,16 @@ The default values of the keywords define the default model setup.
     # RESTART
     write_restart::Bool = output        # also write restart file if output==true?
     restart_path::String = output_path  # path for restart file
-    restart_id::Integer = 1             # run_id of restart file in run????/restart.jld2
+    restart_id::String = "0001"         # run_id of restart file in run????/restart.jld2
```

Comment:
Same here. We could use the format "%04d" under the hood, but also allow for integers that are converted like this.